src/modules/jcms_rest/src/Plugin/rest/resource/JobAdvertListRestResource.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\rest\resource;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\jcms_rest\Response\JCMSRestResponse;
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Provides a resource to get view modes by entity and bundle.
+ *
+ * @RestResource(
+ *   id = "job_advert_list_rest_resource",
+ *   label = @Translation("Job advert list rest resource"),
+ *   uri_paths = {
+ *     "canonical" = "/job-adverts"
+ *   }
+ * )
+ */
+class JobAdvertListRestResource extends AbstractRestResourceBase {
+
+  /**
+   * Responds to GET requests.
+   *
+   * Returns a list of bundles for specified entity.
+   *
+   * @throws \Symfony\Component\HttpKernel\Exception\HttpException
+   *   Throws exception expected.
+   *
+   * @todo - elife - nlisgo - Handle version specific requests
+   */
+  public function get() {
+    $base_query = \Drupal::entityQuery('node')
+      ->condition('status', NodeInterface::PUBLISHED)
+      ->condition('changed', \Drupal::time()->getRequestTime(), '<')
+      ->condition('type', 'job_advert');
+
+    $this->filterShow($base_query);
+    $count_query = clone $base_query;
+    $items_query = clone $base_query;
+    $response_data = [
+      'total' => 0,
+      'items' => [],
+    ];
+    $nodes = [];
+    if ($total = $count_query->count()->execute()) {
+      $response_data['total'] = (int) $total;
+      $this->filterPageAndOrder($items_query);
+      $nids = $items_query->execute();
+      $nodes = Node::loadMultiple($nids);
```

Comment:
I assume this results in 3 queries:
- count
- list ids
- load all these `$nids`

Reply:
Is there a way to log that? We can do it locally with MySQL's log